content/en/docs/reference/labels-annotations-taints/audit-annotations.md

```diff
@@ -72,6 +72,30 @@ violated from the PodSecurity enforcement.
 See [Pod Security Standards](/docs/concepts/security/pod-security-standards/)
 for more information.
 
+## apiserver.latency.k8s.io/etcd
+
+Example: `apiserver.latency.k8s.io/etcd: "4.730661757s"`
+
+This annotation indiactes the measure of latency incurred inside the storage layer,
+it accounts for the time it takes to send data to the etcd and get the complete response back.
```

Comment:
```suggestion
it accounts for the time it takes to send data to etcd and get a complete response back.
```

Reply:
Should we write etcd as `ETCD` here ?just wanted to confirm.

Reply:
See https://etcd.io/ for capitalization